baseledger/ops/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM starport/cli
```

Comment:
just to not forget, will this PR solve logging in containers?

Reply:
Yes